client/src/components/FigmaUI/FButtonMobile/FButtonMobile.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import styled from "styled-components";
+import { colors } from "../../../colors";
+import EVAIcon from "../../UI/EVAIcon/EVAIcon";
+
+interface Props {
+  name: string;
+  t: (a: string, b: string) => void;
+  fill: string;
+  title: string;
+  defaultTitle: string;
+  color: string;
+  disabled: boolean;
```

Comment:
une bonne pratique de nommage des variables est de commencer les booleans par "is" ou "has", donc ici ce serait mieux de mettre isDisabled pour quon comprenne directement que cest un boolean